### PR TITLE
📝 Tabs documentation 

### LINKED ISSUE #2284: Create documentation for Tabs

### PR DESCRIPTION
**Pull request by martalalik**
Resolves #2284

### CODE REVIEW

File: packages/eds-core-react/src/components/Tabs/Tabs.docs.mdx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+### States
+
+<Story id="navigation-tabs--states" />
+
+### Widths
+
+Tabs have:
+<ul>
+    <li><code>minWidth</code> width of Spacer/16 - Medium on each side of the label.</li>
```
Comment (oddvernes) [inline packages/eds-core-react/src/components/Tabs/Tabs.docs.mdx:56]: It is confusing to me what "Spacer/16 - Medium" means

File: packages/eds-core-react/src/components/Tabs/Tabs.docs.mdx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    <li><code>minWidth</code> width of 360px.</li>
```
Comment (oddvernes) [inline packages/eds-core-react/src/components/Tabs/Tabs.docs.mdx:57]: I think this is meant to say fullWidth?

**Review by mimarz — CHANGES_REQUESTED** [review]

File: packages/eds-core-react/src/components/Tabs/Tabs.docs.mdx (hunk 1 above)
Comment (martalalik) [reply packages/eds-core-react/src/components/Tabs/Tabs.docs.mdx:56]: Yes it is 😏

File: packages/eds-core-react/src/components/Tabs/Tabs.docs.mdx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+</ul>
```
[the remark this answers is not in the document]
Comment (martalalik) [reply packages/eds-core-react/src/components/Tabs/Tabs.docs.mdx]: Updated

**Pull request merged**